### DISCUSSION TITLE
Why don't I display the logo and favicon?

### DISCUSSION THREAD

**Discussion by pomponchik** [squidfunk/mkdocs-material#6179] [General]
Hello!

I have a small [project](https://github.com/pomponchik/cantok/tree/develop) and I decided to add `mkdocs` documentation there. I placed the `docs` directory in the root directory of the project. Here's what's inside this folder:

![image](https://i.ibb.co/3zHxTf4/2023-10-12-15-55-02.png)

This is what my `mkdocs.yml` file looks like:

![image](https://i.ibb.co/jVgycrr/2023-10-12-15-22-41.png)

Next, I start the server using the command:

```bash
mkdocs serve -f docs/mkdocs.yml
```

I do this from the root directory of the project.

But! The documentation does not display the logo and favicon. I see logs like these, the request for the favicon does not pass with a 404 error, and there is no request for the logo at all:

```
INFO    -  Building documentation...
INFO    -  Cleaning site directory
INFO    -  Documentation built in 0.35 seconds
INFO    -  [16:20:59] Watching paths for changes: 'docs/docs', 'docs/mkdocs.yml'
INFO    -  [16:20:59] Serving on http://127.0.0.1:8000/
INFO    -  [16:21:01] Browser connected: http://127.0.0.1:8000/
INFO    -  [16:21:01] Browser connected: http://127.0.0.1:8000/
WARNING -  [16:21:02] "GET /assets/favicon.iso HTTP/1.1" code 404
INFO    -  [16:21:02] Browser connected: http://127.0.0.1:8000/
```

Here are the versions of the libraries I use:

```
mkdocs==1.5.3
mkdocs-material==9.2.7
```

What am I doing wrong? Why aren't the logo and favicon displayed? How to fix it?

**Comment by kamilkrzyskow** [comment]
From the image I see you have two `docs` directories, and you put the `assets` into the first instead of the second.

So put them into the second inner `docs` directory.

Also you could rename the first `docs` as something else to not mix them up.

**Reply by pomponchik** [reply]
Okay, I did so, here's a new schema of my files:

![](https://i.ibb.co/6y2BRjm/2023-10-12-17-21-41.png)

Unfortunately, it didn't help. The logo and favicon are not displayed as before. I haven't renamed the `docs` folder yet for the purity of the experiment.

**Reply by alexvoss** [reply]
In the `mkdocs.yml` you seem to have the extension wrong? That would be part of the problem?

**Reply by kamilkrzyskow** [reply]
EDIT:
I missed the file name. Right, so that's all that is left to change after restructuring the files.

---

Weird, perhaps the location at where you run the `mkdocs serve` influences this behaviour, but it shouldn't as it should look up the files relative to the mkdocs.yml file location.

I guess the best path forward is for you to provide a minimal reproduction archive.

https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/

**Reply by pomponchik** [reply]
Please tell me which file extension is correct?

**Reply by alexvoss** [reply]
you have a `.ico` file in you filesystem but the `mkdocs.yml` says `.iso`

**Reply by pomponchik** [reply]
Wow, it really works, my mistake, thank you!
